docs/sources/dashboards/dashboard-public/index.md

```diff
@@ -38,41 +45,55 @@ If you are using Docker, use an environment variable to enable public dashboards
 --env GF_FEATURE_TOGGLES_ENABLE=publicDashboards
 ```
 
-> **Note:** For Grafana Cloud, contact support to have the feature enabled.
+{{% admonition type="Note" %}}
+
+For Grafana Cloud, contact support to have the feature enabled.
+
+{{% /admonition %}}
 
 ## Make a dashboard public
 
-1. Click the sharing icon to the right of the dashboard title.
+1. Click the sharing icon in the dashboard header.
 1. Click the **Public dashboard** tab.
-1. Acknowledge the implications of making the dashboard public by checking all the checkboxes.
+1. Acknowledge the implications of making the dashboard public by selecting all the checkboxes.
 1. Click **Generate public URL** to make the dashboard public and make your link live.
 1. Copy the public dashboard link if you'd like to share it. You can always come back later for it.
 
+Once you've made the dashboard public, a **Public** tag is displayed in the header of the dashboard.
+
 ## Pause access
 
-1. Click the sharing icon to the right of the dashboard title.
+1. Click the sharing icon in the dashboard header.
 1. Click the **Public dashboard** tab.
 1. Enable the **Pause sharing dashboard** toggle.
 
 The dashboard is no longer accessible, even with the link, until you make it shareable again.
 
 ## Revoke access
 
-1. Click the sharing icon to the right of the dashboard title.
+1. Click the sharing icon in the dashboard header.
 1. Click the **Public dashboard** tab.
 1. Click **Revoke public URL** to delete the public dashboard.
 
-The link no longer works. You must create a new public URL as in [Make a dashboard public](#make-a-dashboard-public).
+The link no longer works. You must create a new public URL, as in [Make a dashboard public](#make-a-dashboard-public).
 
 ## Email sharing
 
-> **Note:** Available in [Grafana Enterprise]({{< relref "../../introduction/grafana-enterprise/" >}}) and [Grafana Cloud Pro and Advanced](/docs/grafana-cloud).
+{{% admonition type="Note" %}}
+
+Available in [Grafana Enterprise]({{< relref "../../introduction/grafana-enterprise/" >}}) and [Grafana Cloud Pro and Advanced](/docs/grafana-cloud).
+
+{{% /admonition %}}
 
-Email sharing allows you to share your public dashboard with only specific people via email, instead of having it accessible to anyone with the URL.
+Email sharing allows you to share your public dashboard with only specific people by email, instead of having it accessible to anyone with the URL. When you use email sharing, recipients receive a one-time use link that's valid for **one hour**. Once the link is used, the viewer has access to the public dashboard for **30 days**.
 
 ### Enable email sharing
 
-> **Note:** For Grafana Cloud, contact support to have the feature enabled.
+{{% admonition type="Note" %}}
+
+For Grafana Cloud, contact support to have the feature enabled.
```

Comment:
Same as before with the difference that this feature is also only supported by Pro and Advance as stated above.